task-rest_bold.psyexp

```diff
@@ -153,36 +140,108 @@
         <Param val="" valType="extendedCode" updates="constant" name="Before JS Experiment"/>
         <Param val="" valType="extendedCode" updates="constant" name="Begin Experiment"/>
         <Param val="" valType="extendedCode" updates="constant" name="Begin JS Experiment"/>
-        <Param val="ioServer.getDevice(&quot;tracker&quot;).sendMessage(&quot;hello rs&quot;);&amp;#10;" valType="extendedCode" updates="constant" name="Begin JS Routine"/>
-        <Param val="ioServer.getDevice('tracker').sendMessage(&quot;hello rs&quot;)&amp;#10;" valType="extendedCode" updates="constant" name="Begin Routine"/>
+        <Param val="ioServer.getDevice(&quot;tracker&quot;).sendMessage(&quot;start movie&quot;);&amp;#10;" valType="extendedCode" updates="constant" name="Begin JS Routine"/>
```

Comment:
Does this respond to some advance on theaxonlab/hcph-sops#74 ?

Reply:
Yes, but also connected to https://github.com/TheAxonLab/hcph-sops/issues/74#91 as I have implemented the event file writing for resting state within this context.